src/blr/Snakefile

```diff
@@ -11,7 +11,48 @@ include: "rules/phasing.smk"
 
 
 rule final:
-    input: "reads.1.final.fastq.gz", "reads.2.final.fastq.gz"
+   input:
+        "reads.1.final.fastq.gz",
+        "reads.2.final.fastq.gz",
+        "phasing_stats.txt",
+        "multiqc_report.html"
+
+
+rule fastqc_raw_reads:
+    # Creates fastqc reports from raw read files. Output names are automatically given by fastqc.
+    output:
+        qc_1 = "trimmed.barcoded.1_fastqc.html",
+        qc_2 = "trimmed.barcoded.2_fastqc.html",
+        zip_1 = "trimmed.barcoded.1_fastqc.zip",
+        zip_2 = "trimmed.barcoded.2_fastqc.zip"
+    input:
+        read_1 = "trimmed.barcoded.1.fastq.gz",
+        read_2 = "trimmed.barcoded.2.fastq.gz"
+    log:
```

Comment:
You can use wildcard expansion to reduce the number of argument.

For example
```
qc_1 = "trimmed.barcoded.1_fastqc.html",
qc_2 = "trimmed.barcoded.2_fastqc.html"
```

can become
```
qc = "trimmed.barcoded.{read_nr}_fastqc.html"
```
which will run for read1 and read2 separately.


Or alternatively you can use expansion to run them together.
```
qc = expand("trimmed.barcoded.{read_nr}_fastqc.html", read_nr=(1, 2))
```